library/sync-tasks/src/androidTest/java/org/cru/godtools/sync/task/BaseDataSyncTasksTest.java

```diff
@@ -44,8 +44,8 @@ public void verifyStoreLanguageChangingCode() {
 
         // run test
         mTasks.storeLanguage(mEvents, language(1L, newLocale, false));
-        assertThat(mDao.find(Language.class, originalLocale), nullValue());
-        final Language language = mDao.find(Language.class, newLocale);
+        assertThat(mDao.find(Language.class, originalLocale, originalLocale.getDisplayName()), nullValue());
+        final Language language = mDao.find(Language.class, newLocale, newLocale.getDisplayName());
```

Comment:
display name shouldn't be part of the primary key, so you won't need to include it in the find query